Source/items.h

```diff
@@ -299,7 +299,7 @@ void SpawnRewardItem(int itemid, int xx, int yy);
 void SpawnMapOfDoom(int xx, int yy);
 void SpawnRuneBomb(int xx, int yy);
 void SpawnTheodore(int xx, int yy);
-void RespawnItem(int i, BOOL FlipFlag);
+void RespawnItem(ItemStruct &item, BOOL FlipFlag);

```

Comment:
I think I would prefer `*` over `&` for references as it makes it clear on the calling end that it is by reference.

Reply:
What? But the purpose of references is exactly to abstract the need for specifying pointers in the first place.

Are you really sure about this? Can you point me to supporting documentation on it?

I was actively avoiding raw pointers as much as possible everywhere, so seeing this suggestion doesn't make sense to me.

Reply:
"supporting documentation" on my personal preference? I can write some if you would like that.

> I was actively avoiding raw pointers as much as possible everywhere, so seeing this suggestion doesn't make sense to me.

why?

Reply:
> "supporting documentation" on my personal preference?

No, I meant technical reasons. I'm not basing my decision to use one or the other on "personal preference" here, but based on the fact that references are more robust and lead to less errors than raw pointers (in my understanding).